### DISCUSSION TITLE
Just one subtitle using minipage to show pictures side by side

### DISCUSSION THREAD

**Discussion by RSantos94** [joaomlourenco/novathesis#422] [Q&A]
I'm using Overleaf with latest version from GitHub (6.10.18)

I've been using minipages just like below to show two related pictures side by side, but it seems to be showing only one of the subtitles (the one for image 1)

Below is how I've been using it. (It used to work)

\begin{figure}[htp]
    \centering
    \begin{minipage}[b]{0.45\textwidth}
        \includegraphics[width=\textwidth]{image1.png}
        \caption{Image 1.}
        \label{fig:image1}
    \end{minipage}
    \hfill
    \begin{minipage}[b]{0.45\textwidth}
        \includegraphics[width=\textwidth]{image2.png}
        \caption{Image 2.}
        \label{fig:image2}
    \end{minipage}    
\end{figure}
[log.txt](https://github.com/joaomlourenco/novathesis/files/11063420/log.txt)

**Comment by joaomlourenco** [comment]
Download the original template and see the example on how to put to subfigures side-by-side!
